contact/contact.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="contact-styles.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.1.1/animate.min.css"/>
+  <script src="/landing-page/salon-script.js"></script>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.1.1/animate.min.css"/>
+  <link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <title>Barbershop KRS-Checkpoint1 contact</title>
+</head>
+
+<body>
+  <nav class="nav" id="nav">
+    <ul>
+      <li><a class="active">SAFETY PROTOCOL</a></li>
+      <li class="dropdown"><a href="/landing-page/salon.html">WELCOME TO</a>
+        <ul class="dropdown-content">
+          <li><a href="/about/about.html">ABOUT</a></li>
+          <li><a>SERVICE POLICY</a></li>
+        </ul></li>
+      <li><a>#KRSGALMSQUAD</a></li>
+      <li><a href="/tips-trix/tips.html">TIPS & TRIX</a></li>
+      <li class="dropdown"><a>SERVICES</a>
+        <ul class="dropdown-content">
+          <li><a>HAIR</a></li>
+          <li><a>BRAIDS</a></li>
+          <li><a>LOCS</a></li>
+          <li><a>WAXING</a></li>
+          <li><a>EYELASH EXTENSIONS</a></li>
+          <li><a>BEAUTY</a></li>
+        </ul>
+      </li>
+      <li><a>BOOK NOW</a></li>
+      <li><a>BALENCED</a></li>
+      <li><a href="/contact/contact.html">CONTACT</a></li>
+      <li class="icon">
+        <a href="javascript:void(0);" onclick="myFunction()">
+          <i class="fa fa-bars"></i></a>
+      </li>
+    </ul>
+  </nav>
+    <script src="salon-script.js"></script>
```

Comment:
script tag should go right before the closing </body> tag
`  <script src="salon-script.js"></script>`
`</body>`